lib/command.js

```diff
@@ -735,12 +737,22 @@ Expecting one of '${allowedValues.join("', '")}'`);
    */
   passThroughOptions(passThrough = true) {
     this._passThroughOptions = !!passThrough;
-    if (!!this.parent && passThrough && !this.parent._enablePositionalOptions) {
-      throw new Error('passThroughOptions can not be used without turning on enablePositionalOptions for parent command(s)');
-    }
+    this._checkForBrokenPassThrough(this, this.parent);
     return this;
   }
 
+  /**
+   * @param {Command} command
+   * @param {Command | null} parent
+   * @api private
+   */
+
+  _checkForBrokenPassThrough(command, parent) {
```

Comment:
I think remove the parameters and call the check on the subcommand?

Reply:
Sure. Resolved in [7689506](https://github.com/tj/commander.js/pull/1937/commits/7689506699b9db61a121baea40ba455e8d89c78d).